src/app.py

```diff
@@ -1,13 +1,20 @@
 import pygame
+from Cards import Cards
 
 def main():
      
     pygame.init()
     
-    pygame.display.set_logo()
-    pygame.display.set_caption("Kopi-Cat Uno")
+    programIcon = pygame.image.load('./assets/kopicat.PNG')
```

Comment:
icon should have background removed. But it's suggested (by andrew) that we should do that on a later date.